nautobot/docs/user-guide/platform-functionality/events.md

```diff
@@ -56,3 +56,24 @@ The data payload associated with events of any of the above topics has the follo
 * `prechange` - a dictionary of record attributes and their values *before* the event occurred (or null, in the case of a `nautobot.create` event)
 * `postchange` - a dictionary of record attributes and their values *after* the event occurred (or null, in the case of a `nautobot.delete` event)
 * `differences` - a dictionary with keys `added` and `removed`, each of which is a dictionary of record attributes that changed in the event, providing a convenient alternative to manually comparing the prechange and postchange data.
+
+### Job Events
+* `nautobot.jobs.job.completed` A job gets completed.
+* `nautobot.jobs.job.started` A job started.
+
+The data payload associated with events of any of the above topics has the following keys:
+
+* `job_result_id` - The id of the Job Result.
+* `job_name` - The name of the job.
+* `user_name` - The name of the user account associated with the job.
+* `job_kwargs` - The job data.
+* `job_output` - The output of the completed job (or null, in the case of `nautobot.jobs.job.started`).
+* `einfo` - The stacktrace of job failure (or null, in the case of `nautobot.jobs.job.started` and job competed without a failure).
```

Comment:
Would be preferable IMHO to omit these two keys from the `.started` payload rather than having them present but always null.